builtin/int64.js.mbt

```diff
@@ -497,6 +497,10 @@ pub fn UInt64::to_int(self : UInt64) -> Int {
   MyInt64::from_uint64(self).to_int()
 }
 
+pub fn UInt64::to_double(self : UInt64) -> Double {
+  MyInt64::from_uint64(self).to_double()
```

Comment:
This implementation is not correct. Like the implementations of `UInt64::compare` and `Int64::compare`, the compare operations are different for signed and unsigned integer. In webassembly, there also exists two instructions `i64.trunc_f64_s`/`i64.trunc_f64_u` (which the wasm backend currently uses) to convert double to i64